### PR TITLE
Add new iPhone 10S, 10S Max, and 10R model numbers

### PR DESCRIPTION
**Pull request by AndreyMitsyk**
Added new devices in formFactorMap. Data comes from simulators via calabash-ios gem.

Related PR: https://github.com/calabash/calabash-ios-server/pull/431

### CODE REVIEW

File: Server/Utilities/CBXDevice.m [hunk 1]
```diff
@@ -252,6 +252,11 @@ - (NSDictionary *)formFactorMap {
       @"iPhone10,3" : @"iphone 10",
       @"iPhone10,6" : @"iphone 10",
 
+      // iPhone XS/XS Max/XR - derived from Simulator
```
Comment (jmoody) [inline Server/Utilities/CBXDevice.m]: > // iPhone 8/8+/X - derived from Simulator

@ line 247, let's remove the "derived from Simulator"  - that comment is no longer correct. Some of those model numbers are _not_ from the simulator.

Comment (AndreyMitsyk) [reply Server/Utilities/CBXDevice.m]: Fixed.

File: Server/Utilities/CBXDevice.m [hunk 2]
```diff
@@ -244,14 +244,19 @@ - (NSDictionary *)formFactorMap {
...
       @"iPhone10,4" : @"iphone 6",
       @"iPhone10,5" : @"iphone 6+",
       @"iPhone10,2" : @"iphone 6+",
       @"iPhone10,3" : @"iphone 10",
       @"iPhone10,6" : @"iphone 10",
 
+      // iPhone XS/XS Max/XR - derived from Simulator
+      @"iPhone11,2" : @"iPhone11,2",
+      @"iPhone11,4" : @"iPhone11,4",
+      @"iPhone11,8" : @"iPhone11,8",
```
Comment (jmoody) [inline Server/Utilities/CBXDevice.m]: We are mapping _from_ a model number _to_ a form factor.

The form factor will be one of:

"iphone 10", "iphone 10s max", or "iphone 10r"

Comment (AndreyMitsyk) [reply Server/Utilities/CBXDevice.m]: @jmoody *changed*

**Pull request merged**